doc/site/content/docs/guides/getting-started/widgets-and-gadgets.md

```diff
@@ -102,13 +100,21 @@ function GadgetHandler:add_gadget(self, file)
 end
 
 --- @param delta number
-function callins:Update(self, delta)
+function Update(self, delta)
     for _, gadget in self.gadgets do
         if gadget:Update then
             gadget:Update(delta)
         end
     end
 end
+
+Spring.UpdateCallin('Update')
 ```
 
-This isn't how it's implemented exactly, but it illustrates the concept. The gadgets are loaded from a file and added to the handler's list of gadgets. When a callin is triggered, such as `Update` here, it calls `Update()` on all gadgets that have it overridden.
+This isn't how it's implemented exactly, but it illustrates the concept. The gadgets are loaded from a file and added to the handler's list of gadgets. When a callin is triggered, such as `Update` here, it calls `Update()` on all gadgets that have it overridden (be aware that the available callins vary by endpoint (Intro/Menu/Ui/Rules/Gaia) you can use Script.GetCallInList() to check what is available).
+
```

Comment:
A mention that widgets and gadgets can communicate intra and inter-environment and a ref to https://beyond-all-reason.github.io/RecoilEngine/docs/guides/wupget/